### ISSUE TITLE
Mroonga 11.08

### ISSUE THREAD

**Issue by komainu8** [mroonga/mroonga#450]
We plan to release 11.08 in a few days.

guide: http://mroonga.org/ja/docs/developer/release.html (Japanese)

- [x] NEWS
  - [x] Update news
  - [x] Translate news
- [ ] Prepare Release Note and announcement
  - [x] Blog https://github.com/mroonga/mroonga.github.com
    - [x] English
    - [x] Japanese
  - [ ] Announce on mailing lists
    - [ ] groonga-dev 
    - [ ] groonga-talk
    - [ ] mysql@mysql.gr.jp
    - [ ] Twitter (Japanese/English)
    - [ ] Facebook (Japanese/English)
- [x] Update documentation
  - [x] Update .rst
  - [x] Translate (Update .po)
- [x] Check Travis CI https://travis-ci.org/mroonga/mroonga
- [x] Tagging
- [x] Upload source archives (.tar.gz、.zip)
- [x] Upload packages
  - [x] Debian
  - [x] Ubuntu (launchpad)
  - [x] CentOS
  - [x] Windows
- [x] Upload documentation for http://mroonga.org
- [x] Blog https://github.com/mroonga/mroonga.github.com
  - [x] English
  - [x] Japanese
- [x] Announce on mailing lists
  - [x] groonga-dev 
  - [x] groonga-talk
  - [x] mysql@mysql.gr.jp
- [x] Announce on Twitter
- [x] Announce on Facebook
- [x] Update `base_version`

**Comment by komainu8** [comment]
Announce for Twitter(ja)

```
Mroongaに問題があったため、Mroonga 11.08をリリースしました！ https://mroonga.org/ja/blog/2021/10/06/mroonga-11.08.html Mroongaには、MySQL 5.7で作成されたDBをMySQL 8.0にアップグレードすると、Mroongaがクラッシュする問題がありました。Mroonga 11.08で修正したので、MySQL 8.0 より前のバージョンでMroongaを使用しているユーザーは、Mroonga 11.08以降を使うことを強くおすすめします。
```

Announce for Twitter(en)

```
We found the bug in Mroonga. Therefore, we released Mroonga 11.08! https://mroonga.org/en/blog/2021/10/06/mroonga-11.08.html Mroonga has had a bug that if we upgrade DB created by MySQL 5.7 to MySQL 8.0,
Mroonga crashes. We fixed this bug on Mroonga 11.08. if you were using Mroonga on MySQL 5.7 or previous version, we highly recommended that we use Mroonga 11.08 or later.
```

**Comment by komainu8** [comment]
リリースおわり。
